db/migrate/20170428180327_add_controlled_unlock_required_to_risk_to_others.rb

```diff
@@ -0,0 +1,7 @@
+class AddControlledUnlockRequiredToRiskToOthers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :risks, :controlled_unlock_required, :string, default: "unknown"
```

Comment:
_Defaults_ in the database for values that are controlled at the _form level_ should not be set.

They're generally a bad practice and show how much we're tying the database schema to the business model data.